dev/docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 services:
   waku-node:
-    image: xmtp/node-go@sha256:489f72a4049272989b42032f6fe23aac91cfaf13c93605017d4dce93d21f4993
+    image: xmtp/node-go:latest
     environment:
       - GOWAKU-NODEKEY=8a30dcb604b0b53627a5adc054dbf434b446628d4bd1eccc681d223f0550ce67
     command:
       - --ws
       - --store
+      - --message-db-connection-string=postgres://postgres:xmtp@db:5432/postgres?sslmode=disable
       - --lightpush
       - --ws-port=9001
+      - --wait-for-db=30s
     ports:
       - 9001:9001
+    depends_on:
+      - db
+  db:
+    image: postgres:13
+    environment:
+      POSTGRES_PASSWORD: xmtp
+    ports:
```

Comment:
Don't need to expose this port to the host, which'll lower the chance of port conflicts with other instances locally.